tests/backup/backup_test.go

```diff
@@ -86,6 +88,508 @@ var _ = Describe("{UserGroupManagement}", func() {
 	})
 })
 
+var _ = Describe("{BkpRstrDiffK8sVerSimultaneousDiffNS}", func() {
+
+	var (
+		appList                          = Inst().AppList
+		backupNames                      []string
+		sourceClusterAppsContexts        []*scheduler.Context    // Each Context is for one Namespace which corresponds to one App
+		destinationClusterAppsContexts   []*scheduler.Context    // Each Context is for one Namespace which corresponds to one App
+		backupContexts                   []*BackupRestoreContext // Each Context is for one backup
+		restoreContexts                  []*BackupRestoreContext // Each Context is for one restore
+		restoreLaterContexts             []*BackupRestoreContext // Each Context is for one restore-later
+		preRuleNameList                  []string
+		postRuleNameList                 []string
+		appContexts                      []*scheduler.Context // All app contexts (namespaces) in this loop of the scaling
```

Comment:
What does `in this loop of the scaling` mean?

Reply:
How about removing unnecessary global variables like appContexts?

Reply:
It was redundant; forgot to remove. removed ✅